deploy/11_deploy_risk_manager_v1.ts

```diff
@@ -10,6 +10,8 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const TBTCDepositToken = await deployments.get("TBTCDepositToken")
   const CoveragePool = await deployments.get("CoveragePool")
   const SignerBondsManualSwap = await deployments.get("SignerBondsManualSwap")
+  // TODO: Determine if the ETH<->T pool exists. If there's no pool, then
+  //       this strategy should not be deployed.
```

Comment:
No, everything is happening manually via the proposal. Let's drop it.

Reply:
I removed the code related to the Risk Manager v1